htdocs/themes/xbootstrap/style.css

```diff
@@ -182,6 +188,7 @@ a.navbar-brand.xlogo {
 
 .xoops-blocks {
     margin-bottom: 20px;
+    text-align:justify;
```

Comment:
Personally not a big fan of justified text. Works in print with liberal hyphenation, but browsers are not quite there yet. We'll need to form a consensus on this.